src/library/scala/collection/mutable/ListBuffer.scala

```diff
@@ -407,7 +395,7 @@ class ListBuffer[A]
 @SerialVersionUID(3L)
 object ListBuffer extends StrictOptimizedSeqFactory[ListBuffer] {
 
-  def from[A](coll: collection.IterableOnce[A]): ListBuffer[A] = new ListBuffer[A] ++= coll
+  def from[A](coll: collection.IterableOnce[A]): ListBuffer[A] = new ListBuffer[A].freshFrom(coll)
```

Comment:
Since you never use `freshFrom` to do anything but this, why not just use `from`?  Is this to avoid possible overhead from checking the initialization of the companion object?

Reply:
I don't know what the overhead is from, but back when I was benchmarking #9174, I discovered that `from` had significantly worse performance than `new` (with `++=` I think). if someone has a more reliable benchmarking setup and wants to run it again, that'd be great. that was my primary motivation for using `new` and a private method